tests/test_api.py

```diff
@@ -525,35 +598,95 @@ def test_compile_model_from_params():
     data = load_boston()
     X, y = data.data[:100], data.target[:100]
 
-    losses = ("mean_squared_error", "mean_absolute_error")
+    losses = (losses_module.MeanAbsoluteError, losses_module.MeanSquaredError)
```

Comment:
What value does having two losses provide?

Reply:
If two losses are really needed, I'm not seeing why `pytest.mark.parametrize` can't be used.

Reply:
You're right. I think this grew from a small test, but at this point should be handled by `pytest.mark.parametrize`. I'll implement that.

Reply:
In fact, I see no reason to have two tests. So the simpler thing to do here is just use one. I chose `MeanAbsoluteError`.